app/views/pages/downloads.ctp

```diff
@@ -412,7 +413,38 @@ $this->set('title_for_layout', $pages->formatTitle(__('Download sentences', true
              __('Indicates the self-reported skill levels of members in individual languages.'); ?>
             </dd>  
         </dl>
+        
+        <!-- Users' collections -->
+        <h3><?php __('Users\' collections'); ?></h3>
+        <dl>
+            <dt><?php echo $download_str; ?></dt>
+            <dd>
+            <a href="http://downloads.tatoeba.org/exports/users_sentences.csv">
+             http://downloads.tatoeba.org/exports/users_sentences.csv
+            </a>
+            </dd>
 
+            <dt><?php echo $field_struct_str; ?></dt>
+            <dd>
+                <span class="param"><?php echo $username_str; ?></span>
+                <span class="symbol">[<?php echo $tab_str; ?>]</span>
+                <span class="param"><?php echo $lang_str; ?></span>
+                <span class="symbol">[<?php echo $tab_str; ?>]</span>
+                <span class="param"><?php echo $sent_id_str; ?></span>
+                <span class="symbol">[<?php echo $tab_str; ?>]</span>
+                <span class="param"><?php echo $mark_id_str; ?></span>
+                <span class="symbol">[<?php echo $tab_str; ?>]</span>
+                <span class="param"><?php echo $date_added_str; ?></span>
+                <span class="symbol">[<?php echo $tab_str; ?>]</span>
+                <span class="param"><?php echo $date_last_mod_str; ?></span>
+            </dd>
+
+            <dt><?php echo $file_desc_str; ?></dt>
+            <dd>
+            <?php 
+             __('Contains sentences\' marked by users.'); ?>
```

Comment:
Maybe you could elaborate: what marks is it about?
Maybe you could give an example. For instance, explain the possible values of the mark field.